models/surgical_tool_segmentation/configs/train.json

```diff
@@ -0,0 +1,275 @@
+                    "image",
+                    "label"
+                ],
+                "range_x": "$np.pi",
+                "prob": 0.8,
+                "mode": [
+                    "bilinear",
+                    "nearest"
+                ]
+            },
+            {
+                "_target_": "RandZoomd",
+                "keys": [
+                    "image",
+                    "label"
+                ],
+                "min_zoom": 0.8,
+                "max_zoom": 1.2,
+                "prob": 0.2,
+                "mode": [
+                    "bilinear",
+                    "nearest"
+                ]
+            }
+        ],
+        "preprocessing": {
+            "_target_": "Compose",
+            "transforms": "$@train#deterministic_transforms + @train#random_transforms"
+        },
+        "dataset": {
+            "_target_": "CacheDataset",
+            "data": "$[{'image': i, 'label': l} for i, l in zip(@images, @labels)]",
+            "transform": "@train#preprocessing",
+            "cache_rate": 0.1,
+            "num_workers": 4
+        },
+        "dataloader": {
+            "_target_": "DataLoader",
+            "dataset": "@train#dataset",
+            "batch_size": 8,
+            "shuffle": true,
+            "num_workers": 4
+        },
+        "inferer": {
+            "_target_": "SimpleInferer"
+        },
+        "handlers": [
+            {
+                "_target_": "ValidationHandler",
+                "validator": "@validate#evaluator",
+                "epoch_level": true,
+                "interval": "@val_interval"
+            },
+            {
+                "_target_": "StatsHandler",
+                "tag_name": "train_loss",
+                "output_transform": "$monai.handlers.from_engine(['loss'], first=True)"
+            },
+            {
+                "_target_": "TensorBoardStatsHandler",
+                "log_dir": "@output_dir",
+                "tag_name": "train_loss",
+                "output_transform": "$monai.handlers.from_engine(['loss'], first=True)"
+            }
+        ],
+        "key_metric": {
+            "train_iou": {
+                "_target_": "MeanIoUHandler",
+                "include_background": false,
+                "reduction": "mean",
+                "output_transform": "$monai.handlers.from_engine(['pred', 'label'])"
+            }
+        },
+        "postprocessing": {
+            "_target_": "Compose",
+            "transforms": [
+                {
+                    "_target_": "AsDiscreted",
+                    "argmax": [
+                        true,
+                        false
+                    ],
+                    "to_onehot": [
+                        2,
+                        2
+                    ],
+                    "keys": [
+                        "pred",
+                        "label"
+                    ]
+                }
+            ]
+        },
+        "trainer": {
```

Comment:
In this current setting is it possible to add an option to load a pre-trained checkpoint?